.ci/pytorch/windows/arm64/smoke_test.bat

```diff
@@ -60,6 +60,4 @@ if ERRORLEVEL 1 exit /b 1
 .\simple-torch-test.exe
 if ERRORLEVEL 1 exit /b 1
 
-:end
-set "PATH=%ORIG_PATH%"
```

Comment:
This line was there for a reason. Is that reason no longer valid? If yes, saving the path to `ORIG_PATH` at the beginning fo the script should be removed as well.